src/Symfony/Component/Translation/Translator.php

```diff
@@ -466,11 +466,12 @@ private function getFallbackContent(MessageCatalogue $catalogue)
     private function getCatalogueCachePath($locale)
     {
         $catalogueHash = sha1(serialize(array(
+            'locale' => $this->locale,
```

Comment:
A simple translations cache warmup (that is now available) should clear all cached translations for which the Resources have changed/expired, right?